### PR TITLE
Update Package Metadata

### LINKED ISSUE #57: Update email address and URL in setup.py
# 🐛 Bug Report #

* The `author_email` field listed in `setup.py` is ancient and inaccessible to the devs.  It should be updated to, say, a Trio mailing list.
* The `url` field listed in `setup.py` can probably also be updated to something more modern.

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR updates the `url` and `author_email` package metadata fields to resolve #57 . It also removes the `download_url` field and switches to using the `project_urls` field to provide the same information. Some typos have been fixed in two other fields while I was checking over the metadata.

## 💭 Motivation and context ##

We had some outdated information in the metadata for Python projects that needed to be corrected. Additionally the `download_url` field is discouraged, so I have replaced it with the `project_urls` field. I noticed typos in the `description` and `author` fields and fixed them.

## 🧪 Testing ##

Automated testing passes, and I can see the updated information per:

```console
$ pip show example
Name: example
Version: 0.0.1
Summary: Example Python library
Home-page: https://www.cisa.gov/cybersecurity
Author: Cybersecurity and Infrastructure Security Agency
Author-email: github@cisa.dhs.gov
License: License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
Location: skeleton-python-library/src
Requires: docopt, schema, setuptools
Required-by:
```

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
LGTM! 🐍 📦

File: setup.py [hunk 1]
```diff
@@ -45,16 +45,20 @@ def get_version(version_file):
...
-    # The project's main homepage
-    download_url="https://github.com/cisagov/skeleton-python-library",
+    # Landing page for CISA's cybersecurity mission
+    url="https://www.cisa.gov/cybersecurity",
+    # Additional URLs for this project per
+    # https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls
+    project_urls={
+        "Source": "https://github.com/cisagov/skeleton-python-library",
+        "Tracker": "https://github.com/cisagov/skeleton-python-library/issues",
+    },
```
Comment (jsf9k) [inline setup.py:58]: :+1:

**Review by jsf9k — APPROVED** [review]
Thanks for cleaning this up!

**Review by hillaryj — APPROVED** [review]
💻

**Pull request merged**